copter/source/docs/precision-landing-with-irlock.rst

```diff
@@ -1,23 +1,59 @@
 .. _precision-landing-with-irlock:
 
-=========================================
-Precision Landing and Loiter with IR-LOCK
-=========================================
+============================
+Precision Landing and Loiter
+============================
 
 Overview
 ========
 
-Copter supports Precision Landing using the `IR-LOCK sensor <https://irlock.com/collections/frontpage/products/ir-lock-sensor-precision-landing-kit>`__ and a :ref:`sonar or lidar <common-rangefinder-landingpage>`.
-Using this system, when the vehicle enters LAND mode (and has GPS lock) it is possible to reliably land within 30cm of an IR beacon that is moving at less than 1m/s.
+Copter supports Precision Landing using the `IR-LOCK sensor <https://irlock.com/collections/frontpage/products/ir-lock-sensor-precision-landing-kit>`__ and a :ref:`sonar or lidar <common-rangefinder-landingpage>`, or via `MAVLink LANDING_TARGET messages <https://mavlink.io/en/messages/common.html#LANDING_TARGET>`__ from a companion computer providing position differences from a target such as an `April tag <https://docs.wpilib.org/en/stable/docs/software/vision-processing/apriltag/apriltag-intro.html>`__.
```

Comment:
Might be nice to also add link to this blog: https://discuss.ardupilot.org/t/precision-landing-with-realsense-t265-camera-and-apriltag-part-1-2/48978